apps/frontend/components/BaseModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { Loader2 } from 'lucide-react'
+import React, { Fragment, type ReactNode } from 'react'
+import {
+  AlertDialog,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogHeader,
+  AlertDialogOverlay,
+  AlertDialogTitle
+} from './shadcn/alert-dialog'
+
+interface BaseModalProps {
+  open: boolean
+  handleClose: () => void
+  children?: ReactNode
+  loading?: boolean
+  loadingMessage?: string
+  title?: string
+  description?: string
+  modalBgDarkMode?: boolean
+}
+
+/**
+ *
+ * @remarks
+ * * Use BaseModal Component by creating a new component(which includes 'AlertDialogFooter') that extends BaseModal
+ * * AlertDialogFooter section (Button section) is separated using ConfirmModal component for reusability.
+ */
+export default function BaseModal({
+  open,
+  handleClose,
+  children,
+  loading = false,
+  loadingMessage = '',
+  title = '',
+  description = '',
+  modalBgDarkMode = false
+}: BaseModalProps) {
+  const formattedDescription =
+    description.split('\n').map((line, index) => (
+      <Fragment key={index}>
+        {line}
+        <br />
+      </Fragment>
+    )) ?? ''
```

Comment:
```suggestion
      <p key={index}>
        {line}
      </p>
```
줄바꿈을 `<br />` 대신 p 태그를 사용하는 건 어떨까요? 의미적으로도 좋고 추후 간격 조정(CSS 적용)에도 좋을 것 같아요